docs/TASK.md

```diff
@@ -215,6 +238,19 @@ export const task: Task = {
 
 > Note that you need to rebuild discojs every time you make changes to it (`cd discojs; rm -rf dist/; npm run build`).
 
+## Summary 
+
+- In ```disco/discojs/discojs-core/src/tasks/``` define your new custom task by instanciating a Task object, and define the async function ```model```. You will need to have your model in the .json + .bin format.
```

Comment:
you can omit the parent folder "disco", since (1) the user might have named it differently (2) it is assumed the user is in the project's folder